src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format("ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf) format("ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
```

Comment:
do you need this?

Reply:
Yes, I need this 
The mistake was that I wrote wrong font-weight for this font-fase 
I changed it to 900, so it solves the previous issue with font-weight: 900;